apps/bluetooth/app.js

```diff
@@ -65,7 +65,7 @@ module.exports = function (activity) {
     logger.debug(`speak: ${text}`)
     if (!textIsEmpty(text)) {
       return activity.setForeground().then(() => {
-        if (wifi.getWifiState() === wifi.WIFI_CONNECTED) {
+        if (network.wifiStatus()['state'] === network.CONNECTED) {
```

Comment:
Why not a function without "wifi", just like `network.isConnectedInternet()`

Reply:
I'll fix it.